probe/main.go

```diff
@@ -104,7 +104,8 @@ func main() {
 	}
 
 	var (
-		endpointReporter = endpoint.NewReporter(hostID, hostName, *spyProcs, *useConntrack)
+		revResolver      = endpoint.NewReverseResolver(endpoint.RAddrCacheLen)
+		endpointReporter = endpoint.NewReporter(hostID, hostName, *spyProcs, *useConntrack, revResolver)
```

Comment:
Given cache len is now not configurable, lets roll the initialisation of this in to NewReporter and make RAddrCacheLen 'private'